DAS/README.md

```diff
@@ -2,28 +2,91 @@
 
 *This directory contains scripts to log and test various components of the data data-acquisition-system*
 
-## **Basic Setup and Usage**
+
+
+## Basic Setup and Usage
 These steps assume that you are in the data-acquisition-system root directory.
 1. `cd DAS`
 2. `poetry install`
 3. `poetry shell`
-4. Run any DAS command, eg. `python -m das.mqtt_wireless_logger` (or `python mqtt_wireless_logger.py` as appropriate for your current directory)
+4. Run any DAS command, _e.g._ `python -m das.mqtt_recorder`
+
+## Run Tests
+These steps assume that you have done the basic setup
+1. `pytest`
+
+<br/>
+
+## [MQTT Recorder](/DAS/das/mqtt_recorder.py)
+The command line tool allows for mqtt messages on any topic to be logged and recorded to a csv file. The saved csv files will be stored in the `das/csv_data`. 
+
+### Usage
+```
+# General command
+python -m das.mqtt_recorder [TOPICS] [FLAGS]
+
+# Subscribe to all topics on the V3 bike
+python -m das.mqtt_recorder /v3/#  -v
 
-<br>
+# Subscribe just BOOST and wireless module topics
+python -m das.mqtt_recorder /v3/wireless-module/# power-model/# -v
+```
 
-## **[MQTT Wireless Logger](https://github.com/monash-human-power/data-acquisition-system/blob/master/DAS/das/mqtt_wireless_logger.py)**
-Once started this script will log any data coming in on the MQTT module data and battery channels. The saved csv files will be stored in the `das/csv_data`. 
+| Flag                       |                          Info                           |
```

Comment:
Maybe consider making a column for the default value?

Would be awesome if we could generate this automatically from the `argparse` usage... if only

Reply:
Good idea!